### PR TITLE
composer: log collected txs

### PR DESCRIPTION
**Pull request by itamarreif**
## Summary
This just adds logging for incoming transactions to make debugging easier.

### CODE REVIEW

File: crates/astria-composer/src/searcher/collector.rs [hunk 1]
```diff
@@ -123,13 +123,18 @@ impl Collector {
...
+                        inner: tx.clone(),
                     },
                     Duration::from_millis(500),
                 )
                 .await
             {
-                Ok(()) => {}
+                Ok(()) => {
+                    info!(
+                        transaction.hash = %tx.hash,
```
Comment (noot) [inline crates/astria-composer/src/searcher/collector.rs:134]: define `let hash = tx.hash` above the `match searcher_channel` line, so you don't have to `clone()` the `tx`

Comment (joroshiba) [reply crates/astria-composer/src/searcher/collector.rs:134]: would it make sense to just have the whole log before the match?

This is only logged if sending to the searcher is succesful, some txs might not be logged.

File: crates/astria-composer/src/searcher/collector.rs [hunk 2]
```diff
@@ -123,13 +123,18 @@ impl Collector {
...
+                        chain_id = %chain_id,
+                        "collected transaction with hash `{}` from rollup with chain_id `{}`", tx.hash, chain_id);
```
[the remark this answers is not in the document]
Comment (itamarreif) [reply crates/astria-composer/src/searcher/collector.rs:136]: good point, moved it to before sending to the channel

File: crates/astria-composer/src/searcher/collector.rs (hunk 1 above)
Comment (itamarreif) [reply crates/astria-composer/src/searcher/collector.rs:134]: yeah my bad here, fixed

**Review by joroshiba — APPROVED** [review]

File: crates/astria-composer/src/searcher/collector.rs [hunk 3]
```diff
@@ -119,6 +119,7 @@ impl Collector {
             .await
             .wrap_err("failed to subscribe eth client to full pending transactions")?;
         while let Some(tx) = tx_stream.next().await {
+            debug!(transaction.hash = %tx.hash, "collected transaction with from rollup");
```
Comment (noot) [inline crates/astria-composer/src/searcher/collector.rs:122]:
```suggestion
            debug!(transaction.hash = %tx.hash, "collected transaction from rollup");
```

Comment (noot) [reply crates/astria-composer/src/searcher/collector.rs:122]: do you want to log which rollup (or is there only one)?

**Review by noot — APPROVED** [review]

**Review by SuperFluffy — APPROVED** [review]

**Pull request merged**